.github/workflows/success.sh

```diff
@@ -34,6 +34,7 @@ bin)
     # Create a tag of form v0.0-183-gdf2b162-20191112132344
     rm -rf releasing/out
     git tag "$TAG" || true
+    echo "::set-output name=TAG::$(echo $TAG)"
```

Comment:
I suggest using `echo "TAG=$TAG" >> $GITHUB_ENV` instead. See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable.

Reply:
Done!

Reply:
This didn't seem to work - https://github.com/mithro/verible/actions/runs/403055934

Reply:
See https://github.com/mithro/verible/pull/2